roxie/ccd/ccdlistener.cpp

```diff
@@ -174,7 +174,7 @@ class CascadeManager : public CInterface
         lockQuery.appendf("<control:childlock thisEndpoint='%d' parent='%d'/>", activeIdxes.item(idx), myEndpoint);
         doChildQuery(idx, lockQuery.str(), lockReply);
```

Comment:
Perhaps we could check for lockReply being empty and log that ?

Reply:
yes, but I would prefer to keep the 6.4 change minimal - improving error reporting can go in master